src/views/test/GeneratedCustomers.tsx

```diff
@@ -1,12 +1,59 @@
-import data from '../customer/CustomerListView/data'
 import React, { useState } from 'react';
 import GeneratedTable from './GeneratedCustomersTable'
+import { useFilterCustomerGridDataQuery } from 'src/generated/graphql'
+// import { Customer_Order_By } from 'src/generated/graphql'
 
 const GeneratedCustomers = () => {
-  const [customers] = useState(data);
+  const [sort, setSortCustomers] = React.useState(
+    {
+      name: 'asc'
+    }
+  );
+  const [limit, setLimit] = useState<number>(2);
+  const [page, setPage] = useState<number>(0);
+  const [offset, setOffset] = useState<number>((page) * limit);
+  const [filter, setFilter] = useState<object>({
+    name: {
+      _ilike: "%%"
+    }
+  }
+  )
+
+  const [result] = useFilterCustomerGridDataQuery({
+    // @ts-ignore
+    //TODO order_by type
+    variables: { where: filter, limit, offset, order_by: sort }
+  })
+
+  console.log(result, 'result');
+  const { data, error, fetching } = result;
+
+  const totalCustomers: number = data?.customerAggregate?.aggregate?.count
```

Comment:
Some tables will be huge and it will be not effective to calculate count. I am writing this comment just for thinking:
* trick: sometimes you query N+1 rows (instead N) to know if there is one more page)
* small tables can have theoretically count